Scripts/BulkConversion/SharedScriptLib.ps1

```diff
@@ -0,0 +1,91 @@
+Function New-LogEntry
+{
+Param(
+    [Parameter(Position=0)] [string]  $LogValue,
+    [Parameter(Position=1)] [string]  $Component = "",
+    [Parameter(Position=2)] [int]     $Severity  = 1,
+    [Parameter(Position=3)] [boolean] $WriteHost = $true,
+                            [string]  $Path      = $("C:\Temp\Log"),
+                            [string]  $textcolor = "White"
+)
+    IF(!(Test-path -Path $Path)) {$Scratch = mkdir $Path}
```

Comment:
Should you also validate other parameters, like severity cannot be a negative int, or more than x?

Reply:
Also textcolor probably should't be yellow or red since those are warning and error colors respectively.

Reply:
Added Parameter validations to limit what values can be passed in.